### DISCUSSION TITLE
In HttpException response showing status code at the beginning.

### DISCUSSION THREAD

**Discussion by mithun2003** [fastapi/fastapi#11591] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
raise HTTPException(status_code=404, detail="No transaction found for your request")
```

### Description

in fastapi httpexception i get exception like this "404: No transaction found for your request" how to avoid that status code at the beginning of the http exception response

### Operating System

Windows

### FastAPI Version

0.109.1

### Pydantic Version

2.0.3

### Python Version

3.11

**Comment by YuriiMotov** [comment, marked as answer]
You can create your own exception class, inherit it from `HTTPException` and redefine `__str__` method as you need:
```
class MyHTTPException(HTTPException):
    def __str__(self):
        return self.detail
```

Then use `MyHTTPException` instead of `HTTPException`
